tests/validation/tests/v3_api/common.py

```diff
@@ -2084,15 +2084,29 @@ def readDataFile(data_dir, name):
         return f.read()
 
 
-def set_url_password_token(rancher_url, server_url=None):
+def set_url_password_token(rancher_url, server_url=None, version="oldUI"):
```

Comment:
Why are we setting version to "oldUI" by default? We may not need this argument at all if we just reference `RANCHER_SERVER_VERSION` directly. (Since we probably want this behavior for all rancher versions >= 2.6.0)

Reply:
I took out the text of `oldUI` as that isn't important, but RANCHER_SERVER_VERSION isn't available in common.py.

Reply:
Ah I didn't realize we only defined that in `test_custom_host_reg.py`, LGTM then. Though I wonder if we should move `RANCHER_SERVER_VERSION` into common.py